keycloak/keycloak-admin-ui
# Client add executors

## Motivation
1. Saving changed client profiles.
2. Reloading client profiles.
3. Deleting newly created client profile.
4. Showing "Executors" text, "Add Executor" button and Action dropdown on new client form when a new client is created.
5. Adding cypress tests.

## Verification Steps
1. Go to Realm Settings > Client policies > Profiles > Json editor 
2. Change one of the existing profiles, click "Reload" btn and verify that original profiles display
2. Manually add a new profile or change one of non-global profiles, click save btn and verify that client was successfully saved in the table list (formView tab)
3. Go to Realm Settings > Client policies > Profiles >
4. Click "Create new profile" btn and complete all form fields, click "Save", verify that: "Executors" text, "Add Executor" button and Action dropdown are now displaying. From "Action" dropdown select "Delete this client profile", select "Delete" on the delete modal and verify that the client is not listed in the clients table (formView tab)

## Checklist:
